front/lib/api/assistant/global_agents.ts

```diff
@@ -133,7 +133,7 @@ function _getDefaultSearchActionForGlobalAgent({
     sId: agentSid + "-search-action",
     type: "websearch_configuration",
     name: DEFAULT_WEBSEARCH_ACTION_NAME,
-    description: `Search the web for more information.`,
+    description: `Perform a web search. Do not use for common knowledge questions.`,
```

Comment:
Let's move to the anthropic specific meta prompt vs the tool description

Reply:
Indeed much better